inc/c_util/constbuffer.h

```diff
@@ -57,6 +57,8 @@ MOCKABLE_INTERFACE(constbuffer,
     /*this creates a new constbuffer from a memory area*/
     FUNCTION(, CONSTBUFFER_HANDLE, CONSTBUFFER_Create, const unsigned char*, source, uint32_t, size),
 
+    FUNCTION(, CONSTBUFFER_HANDLE, CONSTBUFFER_CreateWithOffsetAndSize, const unsigned char*, source, uint32_t, offset, uint32_t, size),
```

Comment:
>CONSTBUFFER_CreateWithOffsetAndSize [](http://example.com/codeflow?start=35&length=35)

Clearly missing specs, so I will stop and wait for the specs to come in.

Reply:
Ah, I got it this is only here for the spike that you have in the main repo, so that is why it does not have specs/tests :-).

Did not know the context until I saw the other one and read all mails.